### PR TITLE
Developer center integration (II) - Examples (II) 

### PR DESCRIPTION
**Pull request by VictorVelarde**
- [x] source
- [x] style
- [x] dataview
- [x] widget
- [x] interactivity

Finish the work on examples, initiated on https://github.com/CartoDB/web-sdk/pull/70

### CODE REVIEW

**Comment by VictorVelarde** [comment]
Acceptance can be made on vercel, although we will probably have a better review on dev-center

File: examples/style/sizeContinuous.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            latitude: 50.2672297,
+            zoom: 2.5
+          }
+        });
+
+        /*
+          Size values are selected by default, but of course they can be further configured.
+        */
+        const style = carto.viz.style.sizeContinuous('total_pop');
+        const tempLayer = new carto.viz.Layer('ca_measures', style);
```
Comment (borja-munoz) [inline examples/style/sizeContinuous.html:76]: Name for the layer variable could be changed to better reflect the content

File: examples/widget/formulaWidget.html [hunk 2]
```diff
@@ -42,30 +48,31 @@
...
           <!-- panel -->
-          <div class="panel as-panel as-panel--top as-panel--right as-bg--ui-01">
+          <div id="panel" class="as-panel as-panel--top as-panel--right as-bg--ui-01">
             <div class="as-panel__element as-p--16 as-body">
               <div class="as-container as-container--scrollable">
                 <section class="as-box">
                   <h1 class="as-title">Formula widget</h1>
                   <h4 class="as-subheader as-mb--12">
-                    Display values from a Formula DataView aggregation
+                    Display values from a Formula aggregation width
```
Comment (borja-munoz) [inline examples/widget/formulaWidget.html:57]: With

File: examples/widget/formulaWidget.html [hunk 3]
```diff
@@ -42,30 +48,31 @@
...
+                    <em>carto.viz.widget.Formula</em>, In this case, it is the MAX value of estimated population
                   </h4>
                 </section>
                 <section class="as-box">
+                  <!-- An airship widget for the 'global' data -->
                   <as-formula-widget
                     id="formula-global"
-                    class="as-p--16"
                     heading="Total Estimated Population (global)"
```
Comment (borja-munoz) [inline examples/widget/formulaWidget.html:65]: It's not the total but the max population

File: examples/widget/formulaWidget.html [hunk 4]
```diff
@@ -42,30 +48,31 @@
...
+                    description="All countries"
                   >
                   </as-formula-widget>
                 </section>
                 <section class="as-box">
+                  <!-- An airship widget for the 'viewport' data -->
                   <as-formula-widget
                     id="formula-viewport"
-                    class="as-p--16"
                     heading="Total Estimated Population (viewport)"
```
Comment (borja-munoz) [inline examples/widget/formulaWidget.html:74]: It's not the total but the max population

File: examples/widget/histogramWidget.html [hunk 5]
```diff
@@ -52,18 +58,21 @@
...
-                    currently visible. It includes a filtering feature out of the box
+                    Create a histogram widget, aggregating values in bins, with
+                    <em>carto.viz.widget.Histogram</em>
                   </h4>
                 </section>
                 <section class="as-box">
+                  <!-- An airship widget for the 'global' data -->
                   <as-histogram-widget
                     id="histogram-global"
                     heading="Remote Estimated Population (global)"
```
Comment (borja-munoz) [inline examples/widget/histogramWidget.html:69]: I would remove "Remote"

File: examples/style/icon.html [hunk 6]
```diff
@@ -72,7 +59,14 @@ <h4 class="as-subheader as-mb--12">
         carto.auth.setDefaultCredentials({ username: 'public' });
         const deckMap = carto.viz.createMap();
 
-        const style = carto.viz.style.icon('./boat.png');
+        /*
+          In this case this image is just a .png as a relative url (you can use of course an absolute url...)
+          and its size is specified in pixels
+        */
+        const style = carto.viz.style.icon('./boat.png', {
```
Comment (borja-munoz) [inline examples/style/icon.html:66]: I would use a different layer/icon combination. The world_ports layer has too many features within a huge extension/bounds and the icons are overlapped. The icon details are not appreciated at 24x24 size. Maybe we can use a layer with a smaller extension and a more simple/colorful icon with a greater size.

**Comment by borja-munoz** [comment]
Nice (and huge) PR!

**Review by borja-munoz — APPROVED** [review]
Nice (and huge) PR!

File: examples/style/icon.html (hunk 6 above)
Comment (VictorVelarde) [reply examples/style/icon.html:66]: Agree, I'm gonna merge this and prepare a smaller PR with it. Suggestions are welcome!

**Comment by VictorVelarde** [comment]
Thx for the review!

**Comment by VictorVelarde** [comment]
Ups, I have your suggestions locally, but I didn't push them 😄 
I will in the next PR

**Pull request merged**